### DISCUSSION TITLE
Building rptproj leaves build.obj file

### DISCUSSION THREAD

**Discussion by jrmoreno1** [dotnet/msbuild#8355] [Q&A]
Is there a parameter or property that can be set that will prevent this file from being left behind once the build is finished? If it’s not necessary for some other purpose, could it just be deleted?

**Comment by GangWang01** [comment]
The Microsoft Reporting Services extension has its own target and task implemented, not implemented by msbuild. Its own task saves build results info in build.obj file. The question should be asked there https://marketplace.visualstudio.com/items?itemName=ProBITools.MicrosoftReportProjectsforVisualStudio2022&ssr=false#qna.
